src/components/json/worldData.js

```diff
@@ -119,6 +120,21 @@ const data = [
 		linkedin: "https://linkedin.com/in/iamKarthikBK",
 		twitter: "https://twitter.com/iamKarthikBK",
 		portfolio: "https://www.nohello.com",
+	},
+	{
+		longitude: 47.54352832810872,
```

Comment:
Hey @MuhammadHassanRazaa 
Could you please verify the co-ordinates? Seems to me like you swapped the lat and long by mistake. The location seems to be in UAE currently.

Reply:
I figured this out because your chapter is in Hungary.

Reply:
Oh yeah, just made the corrections. Thanks for pointing out.